### DISCUSSION TITLE
Language server with LSPConfig doing nothing

### DISCUSSION THREAD

**Discussion by ArkasisProjects** [mfussenegger/nvim-jdtls#368] [Q&A]
Hey, this is my first time asking a question, so please be gentle :D.

I got a problem with my java language server. Everything works fine, its spawning and no error message does show up, but there is nothing happening. Neither autocomplete nor any other tool is working.
I installed the language server with nvim-lsp-installer and just called the setup function.

Any ideas?

**Comment by mfussenegger** [comment]
nvim-jdtls doesn't have a `setup` method, so not sure what you did.
You can try following the readme step by step: https://github.com/mfussenegger/nvim-jdtls#language-server-installation 

And if that still doesn't work check the troubleshooting section: https://github.com/mfussenegger/nvim-jdtls#troubleshooting
